modules/oadp-self-service-phases.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-self-service-phases_{context}"]
+= {oadp-short} Self-Service backup and restore phases
+
+The `status.phase` field of a `NonAdminBackup` (NAB) CR and a `NonAdminRestore` (NAR) CR provide an overview of the current state of the CRs. Review the values for the NAB and NAR phases in the following table.
+
+The phase of the CRs only progress forward. Once a phase transitions to the next phase, it cannot revert to a previous phase.
+
+.Phases
+|===
+|*Value* |*Description*
+|New|A creation request of the NAB or NAR CR is accepted by the NAC, but it has not yet been validated by the NAC.
+|BackingOff|NAB or NAR CR is invalidated by the NAC CR because of an invalid `spec` of the NAB or NAR  CR. 
+
+The namespace admin user can update the NAB or NAR `spec` to comply with the policies set by the administrator. After the namespace admin user edits the CRs, the NAC reconciles the CR again.
+|Created|NAB or NAR CR is validated by the NAC, and the `Velero` backup or restore object is created.
+|Deletion|NAB or NAR CR is marked for deletion. The NAC deletes the corresponding `Velero` backup or restore object. When the `Velero` object is deleted, the NAB or NAR CR is also deleted.
```

Comment:
Nit. I am assuming that the values are the outputs of some sort of status message in the CLI. If so, the values should be in backticks, similar to the previous table.